README.md

```diff
@@ -29,157 +33,74 @@ Sonarcloud status:
 [![Technical Dept](https://sonarcloud.io/api/project_badges/measure?project=org.itsallcode%3Aopenfasttrace&metric=sqale_index)](https://sonarcloud.io/dashboard?id=org.itsallcode%3Aopenfasttrace)
 [![Vulnerabilities](https://sonarcloud.io/api/project_badges/measure?project=org.itsallcode%3Aopenfasttrace&metric=vulnerabilities)](https://sonarcloud.io/dashboard?id=org.itsallcode%3Aopenfasttrace)
 
+**Quick Links**
+
 * [Blog](https://blog.itsallcode.org/)
-* [Contributing guide](CONTRIBUTING.md)
-* [OpenFastTrace stories](https://github.com/itsallcode/openfasttrace/wiki/OFT-Stories)
 * [User Guide](doc/user_guide.md)
+* [OpenFastTrace stories](https://github.com/itsallcode/openfasttrace/wiki/OFT-Stories)
+* [Developer Guide](doc/developer_guide.md)
+* [Contributing guide](CONTRIBUTING.md)
 * [Command Line Usage](doc/usage.txt)
 * [Design](doc/design.md)
 * [System Requirements](doc/system_requirements.md)
 
-## Download
+## Using OpenFastTrace
+
+If you want to use OFT, you have the choice between using it as part of your build process &mdash; typically with Maven or Gradle. Or you can run OFT from the command line.
+
+Check the [user guide](doc/user_guide.md) for detailed information on how to use OpenFastTrack.
 
-Download the executable jar at [jcenter](https://jcenter.bintray.com/org/itsallcode/openfasttrace/):
+## Getting OpenFastTrace
 
-* [openfasttrace-2.1.0.jar](https://jcenter.bintray.com/org/itsallcode/openfasttrace/2.1.0/openfasttrace-2.1.0.jar)
+### Getting Pre-Built Packages
 
-### Maven
+Pre-Built JAR files (called `openfasttrace-2.2.0.jar`) are available from the following places:
 
-To use OpenFastTrace as a dependency in your maven project add this to your `pom.xml`:
+* [Maven Central](https://repo1.maven.org/maven2/org/itsallcode/openfasttrace/2.2.0/openfasttrace-2.2.0.jar)
+* [JCenter](https://jcenter.bintray.com/org/itsallcode/openfasttrace/2.2.0/openfasttrace-2.2.0.jar)
+* [GitHub](https://github.com/itsallcode/openfasttrace/releases/download/2.2.0/openfasttrace-2.2.0.jar)
+ 
+#### Getting OFT via Maven
+
+To use OpenFastTrace as a dependency in your [Maven](https://maven.apache.org) project add this to your `pom.xml`:
 
 ```xml
 <dependencies>
     <dependency>
         <groupId>org.itsallcode</groupId>
         <artifactId>openfasttrace</artifactId>
-        <version>2.1.0</version>
+        <version>2.2.0</version>
         <scope>compile</scope>
     </dependency>
 </dependencies>
 ```
 
-### Gradle
+### Getting OFT via Gradle
 
-To use OpenFastTrace as a dependency in your gradle project:
+To use OpenFastTrace as a dependency in your [Gradle](https://gradle.org/) project:
 
 ```groovy
-repositories {
-    jcenter()
-}
 dependencies {
-    compile "org.itsallcode:openfasttrace:2.1.0"
+    compile "org.itsallcode:openfasttrace:2.2.0"
 }
```

Comment:
We should remove this section. Using oft as a library is not the typical use case.